postgres/datadog_checks/postgres/statements.py

```diff
@@ -250,8 +251,34 @@ def _load_pg_stat_statements(self):
 
             return []
 
+    def _emit_pg_stat_statements_metrics(self):
+        try:
+            rows = self._execute_query(
+                self._check._get_db(self._config.dbname).cursor(cursor_factory=psycopg2.extras.DictCursor),
+                PG_STAT_STATEMENTS_COUNT_QUERY,
+            )
+            count = 0
+            if rows:
+                count = rows[0][0]
+            self._check.count(
+                "postgresql.pg_stat_statements.max",
+                self._check.pg_settings.get("pg_stat_statements.max"),
```

Comment:
Should explicitly specify a fallback value here in case it's not loaded. I think `0`? Else it'll be `None`.

Reply:
Yeah, I double checked the `count()` implementation and it does return early if `value` is None, although now that you point it out, I'll add it so it reads nicer, ty!